### PR TITLE
docs: update dpdkstats documentation

### PR DESCRIPTION
**Pull request by maryamtahhan**
Update the dpdkstats documentation to state that it shouldn't be used with
OVS-with-DPDK. The Open vSwitch plugins is what should be used.

Signed-off-by: Maryam Tahhan <maryam.tahhan@intel.com>

### CODE REVIEW

**Comment by maryamtahhan** [comment]
not sure why CI failed - as I didn't make any code changes

**Comment by mrunge** [comment]
Thank you for this clarification @maryamtahhan .

Not sure if that's the right way here, but I would propose the patch to master and then cherry-pick it to earlier branches, in order to prevent regressions.


The failure looks quite suspicious, but completely unrelated to the change itself:

/usr/bin/ld: cannot find -lssl
/usr/bin/ld: cannot find -lcrypto
collect2: error: ld returned 1 exit status
Makefile:3743: recipe for target 'notify_email.la' failed
make[3]: *** [notify_email.la] Error 1

**Comment by mrunge** [comment]
Oh, I just saw, the commit https://github.com/collectd/collectd/pull/2613/commits/19a024b9e2711c6480abe0b7d61af70dbba2242f references collectd 5.8, but the patch itself is for 5.7

**Comment by maryamtahhan** [comment]
@octo @mfournier any idea if this is a 5.7.0 build issue ?
Thanks
Maryam

File: docs/BUILD.dpdkstat.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+that don't expose stats in any other way than the DPDK xstats API.
+
 **Data Plane Development Kit** (DPDK) is a set of drivers and libraries for fast
-packet processing.
+packet processing. Please note that this plugin is a polling based plugin rather
+than an events based plugin (using it will drive up core utilization on a system).
+
+**PLEASE DO NOT USE THIS PLUGIN FOR OVS-DPDK**. dpdkstat is really for DPDK
+applications that have no other way of exposing stats. For OVS or OVS-with-DPDK the
+Open vSwitch plugins available in collectd Version 5.8.0 should be used for
```
Comment (rpv-tomsk) [inline docs/BUILD.dpdkstat.md]: Hi, Maryam!

I'm a bit confused by the capital letter in 'Version'. Is I'm right that a dot is missing in this line?

Comment (maryamtahhan) [reply docs/BUILD.dpdkstat.md]: it's a typo, I will fix it/remove it

**Comment by rpv-tomsk** [comment]
Thanks for the work!

**Pull request merged**